pkg/convert/go_messages.go

```diff
@@ -0,0 +1,67 @@
+package convert
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+	"golang.org/x/text/message/pipeline"
+)
+
+func FromGoMessages(m model.Messages) ([]byte, error) {
+	pipelineMsg := messagesToPipeline(m)
+
+	msg, err := json.Marshal(pipelineMsg)
+	if err != nil {
+		return nil, fmt.Errorf("failed pipeline.Messages marshaling: %w", err)
```

Comment:
Advisory. Describe the intent when wrapping error messages. No need to emphasize something failed. It's an error message, we know something failed. It makes it more succinct and clearer. E.g. `encode pipeline.Messages to JSON: %w`.